### PR TITLE
Assignment3

### PR DESCRIPTION
**Pull request by lchansen**
Unable to finish ex02 in time, it compiles at least 👍 
Lucas Hansen
4666-0800

### CODE REVIEW

File: ex01.exs [hunk 1]
```diff
@@ -26,8 +26,23 @@ defmodule Ex01 do
         2 is the program well laid out,  appropriately using indentation,
           blank lines, vertical alignment
   """
-  
+
   def counter(value \\ 0) do
+    receive do
+      {:next, from} ->
+        send(from, {:next_is, value})
+    end
```
Comment (pragdave) [inline ex01.exs:34]: No penalty, but my old, tired eyes find

~~~
counter value + 1
~~~

harder to parse than

~~~
counter(value + 1)
~~~

File: ex03.exs [hunk 2]
```diff
@@ -62,9 +62,19 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    « your code here »
+    n = collection |> Enum.count |> div(process_count)
+
```
Comment (pragdave) [inline ex03.exs:66]: I like this. 

(You don't need the module name on calls to functions in the same module)

**Comment by pragdave** [comment]
> Unable to finish ex02 in time, it compiles at least 👍

Ship it!

It's a shame, because you aced 1 and 3.

General comments:

Good strategy: switching to ex03, which had higher grades, when you were running out of time.

Bad strategy: running out of time...

File: ex01.exs (hunk 1 above)
Comment (lchansen) [reply ex01.exs:34]: fair 😝

**Pull request closed without merging**